react/features/whiteboard/actions.any.ts

```diff
@@ -0,0 +1,28 @@
+import { IReduxState, IStore } from '../app/types';
+
+import { setWhiteboardOpen } from './actions';
+import { isWhiteboardAllowed, isWhiteboardOpen, isWhiteboardVisible } from './functions';
+
+
+/**
+ * API to toggle the whiteboard.
+ *
+ * @param {Object} state - The redux state.
+ * @returns {Function}
+ */
+export function toggleWhiteboard(state: IReduxState) {
```

Comment:
You shouldn't need to pass the whole state as a param, it should be available as the second param in the return function below.

Reply:
Got it.